### PR TITLE
CRAN updates and clean

### PR DESCRIPTION
**Pull request by mattsecrest**
- [] clean up and prepare for CRAN submission
- [] QC vignettes and look for consistency and clarify

### CODE REVIEW

**Comment by mattsecrest** [comment]
Getting an error on winbuilder
[00check.log.txt](https://github.com/Genentech/psborrow2/files/14641720/00check.log.txt)

**Comment by gravesti** [comment]
@mattsecrest I think I've fixed the failing test and the note about the URLs. I've uploaded again to winbuilder

**Comment by mattsecrest** [comment]
- [ ] load `library(cmdstanr)` in vignettes
- [ ] throw errors for cmdstanr stuff
- [ ] getter for stan models

File: DESCRIPTION [hunk 1]
```diff
@@ -20,18 +21,26 @@ Authors@R: c(
...
+   (e.g. sample size, covariates) in a unified way. Third, 'psborrow2' provides
+   a set of functions to generate data for simulation studies, and also allows
+   the user to specify their own data generation process.
 URL: https://github.com/Genentech/psborrow2, https://genentech.github.io/psborrow2/index.html
 BugReports: https://github.com/Genentech/psborrow2/issues
-License: Apache License (>= 2)
+License: Apache License 2.0
 Encoding: UTF-8
 Depends:
     R (>= 3.5.0)
```
Comment (gravesti) [inline DESCRIPTION:46]: I think the `\(x)` feature came in 4.1
```suggestion
    R (>= 4.1.0)
```

**Comment by mattsecrest** [comment]
@gravesti good catch on the \(x) functionality. Anything else before we submit? I figure we can build and submit from this branch then merge when accepted?

**Review by gravesti — APPROVED** [review]

**Pull request merged**